web/sites/default/config/lightning_scheduler.settings.yml

```diff
@@ -1 +1 @@
-time_step: 3600
```

Comment:
Here is the other override that was thrown off. This should be 3600. The good news is I think these were probably from running `drush update:lightning` at some point. Let's chat about this real quick and figure out the source before you fix.